docs/download_files.md

```diff
@@ -162,9 +162,9 @@ This file will contain fields equivalent to those found in the `single_cell_meta
 Metadata for all samples on the Portal is available to download separately from gene expression data downloads.
 Each project page has an option to download metadata for all of its samples as a single zip file containing the `metadata.tsv` file and a `README.md` file.
 Project-specific metadata will contain all columns listed in [the above table](#metadata) and any additional project-specific columns, such as treatment or outcome.
-<!--
+
 Additionally, the metadata for all samples on the Portal is available and contains the metadata for all samples available on the Portal.
```

Comment:
```suggestion
Additionally, the metadata for all samples on the Portal is available for download as a TSV file and contains the metadata for all samples available on the Portal.
```

Reply:
Oof this sentence was a lot of repetition, so I reworded it in https://github.com/AlexsLemonade/scpca-docs/pull/320/commits/c61a0a00d03cfd711d7ebd56aca58beaf502261e.

Reply:
yeah, https://github.com/AlexsLemonade/scpca-docs/commit/c61a0a00d03cfd711d7ebd56aca58beaf502261e is better!